src/cursive.rs

```diff
@@ -77,6 +77,13 @@ impl<F: FnOnce(&mut Cursive) -> () + Send> CbFunc for F {
     }
 }
 
+#[cfg(feature = "crossterm-backend")]
```

Comment:
You'll have to update all other `impl Default` block to add a `not(feature = "crossterm-backend")` in their `cfg` conditions, otherwise you could get overlapping implementations.